MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -102,21 +109,24 @@ public class MediaEncoder : IMediaEncoder, IDisposable
         /// </summary>
         public void SetFFmpegPath()
         {
+            _bestLocation = FFmpegLocation.NotFound;
+            _bestPath = null;
+            _bestVersion = null;
+
             // 1) Custom path stored in config/encoding xml file under tag <EncoderAppPath> takes precedence
             if (!ValidatePath(_configurationManager.GetConfiguration<EncodingOptions>("encoding").EncoderAppPath, FFmpegLocation.Custom))
             {
                 // 2) Check if the --ffmpeg CLI switch has been given
                 if (!ValidatePath(_startupOptionFFmpegPath, FFmpegLocation.SetByArgument))
                 {
                     // 3) Search system $PATH environment variable for valid FFmpeg
-                    if (!ValidatePath(ExistsOnSystemPath("ffmpeg"), FFmpegLocation.System))
-                    {
-                        EncoderLocation = FFmpegLocation.NotFound;
-                        _ffmpegPath = null;
-                    }
+                    ValidatePath(ExistsOnSystemPath("ffmpeg"), FFmpegLocation.System);
```

Comment:
Given that we're now defaulting to the "best unrecommended" version this should be logged in a warning.

Reply:
done